### PR TITLE
added files

### PR DESCRIPTION
**Pull request by Rehan-Khan-Engineer**

### CODE REVIEW

File: tasks-by-users/Rehan-Khan-Engineer/04-Text-typography-css/style.css [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  background: #000;
+}
+
+.container img {
+  position: absolute;
+  top: 0;
+  left: 0;
+  height: 100vh;
+  width: 100%;
+  object-fit: cover;
```
Comment (Abbas4tech) [inline tasks-by-users/Rehan-Khan-Engineer/04-Text-typography-css/style.css:19]: Group Similar Properties @Rehan-Khan-Engineer

File: tasks-by-users/Rehan-Khan-Engineer/04-Text-typography-css/index.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>RK Practices</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="container">
+      <img src="face.jpg" />
+      <img src="text.jpg" />
```
Comment (Abbas4tech) [inline tasks-by-users/Rehan-Khan-Engineer/04-Text-typography-css/index.html:13]: Alt ? @Rehan-Khan-Engineer

**Review by Abbas4tech — CHANGES_REQUESTED** [review]
Some Changes are required. Please update this!

File: tasks-by-users/Rehan-Khan-Engineer/04-Text-typography-css/style.css [hunk 3]
```diff
@@ -0,0 +1,24 @@
+* {
+  margin: 0;
+  padding: 0;
```
Comment (Abbas4tech) [inline tasks-by-users/Rehan-Khan-Engineer/04-Text-typography-css/style.css:3]: Add `box-sizing: border-box;` @Rehan-Khan-Engineer

**Review by Abbas4tech — CHANGES_REQUESTED** [review]

File: tasks-by-users/Rehan-Khan-Engineer/04-Text-typography-css/style.css [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+}
+
+.container {
+  position: relative;
+  height: 100vh;
+  width: 100%;
```
Comment (shaikh-7abish) [inline tasks-by-users/Rehan-Khan-Engineer/04-Text-typography-css/style.css:9]: Give this `viewport width` as you had given to height

Comment (Rehan-Khan-Engineer) [reply tasks-by-users/Rehan-Khan-Engineer/04-Text-typography-css/style.css:9]: Done @shaikh-7abish

File: tasks-by-users/Rehan-Khan-Engineer/04-Text-typography-css/style.css (hunk 1 above)
Comment (Rehan-Khan-Engineer) [reply tasks-by-users/Rehan-Khan-Engineer/04-Text-typography-css/style.css:19]: Done @Abbas4tech

File: tasks-by-users/Rehan-Khan-Engineer/04-Text-typography-css/style.css (hunk 3 above)
Comment (Rehan-Khan-Engineer) [reply tasks-by-users/Rehan-Khan-Engineer/04-Text-typography-css/style.css:3]: Done @Abbas4tech

File: tasks-by-users/Rehan-Khan-Engineer/04-Text-typography-css/index.html (hunk 2 above)
Comment (Rehan-Khan-Engineer) [reply tasks-by-users/Rehan-Khan-Engineer/04-Text-typography-css/index.html:13]: Done @Abbas4tech

**Review by shaikh-7abish — APPROVED** [review]

**Review by shaikh-7abish — APPROVED** [review]

**Review by Abbas4tech — APPROVED** [review]

**Review by Rehmatulla-Khan — APPROVED** [review]

**Pull request merged**